src/lib/consts.tsx

```diff
@@ -106,3 +106,22 @@ export const COLLECTIONS = [
     type: "ERC1155",
   },
 ]
+
+export const zoraUniversalMinterAddress = {
+  1: "0x308E190d70c7d1C6Ed569554bCe73Dc3F4ad359A",
+  5: "0x1Eb7Bf3a08784D7cB08CC2AE1448012C0c02bDa2",
+  10: "0x97eb05B8db496B12244BCcf17CF377d00a99b67a",
+  420: "0x39C51a7957651ea176733F19125BD9c253894D6F",
+  424: "0xF82286760a953D2Bad7D6F2F0da458Ac20f955D3",
+  999: "0xD9bC36841C259f07924e73cF08d5a2c92d53639B",
+  8453: "0x308E190d70c7d1C6Ed569554bCe73Dc3F4ad359A",
+  42161: "0xC6899816663891D7493939d74d83cb7f2BBcBB16",
+  58008: "0xE9BaDfb9a1658cDF67D8c4631a7f22610C013319",
+  81457: "0xC6899816663891D7493939d74d83cb7f2BBcBB16",
+  84531: "0x418B87c2C9579d27FC3D66605545AB9889737E60",
+  421614: "0x308E190d70c7d1C6Ed569554bCe73Dc3F4ad359A",
+  7777777: "0xF482C51346f3c77673dc619F243Eb8B09E9A954E",
+  11155111: "0x0ef82DaB14798E63F1B99479Ba689e3f6A6fEb6C",
+  168587773: "0xa718BD919eeA529ac75EEf2cf33363AF211f09f4",
+  999999999: "0xD662FB0fB00261C039441EF49Dbab154d7c533bD",
+}
```

Comment:
Why is the value for `zoraUniversalMinterAddress` not being referenced from the Zora SDK???